### PR TITLE
Add more fixtures and snapshot tests for dashboard

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: src/components/Dashboard/Details/tests/DetailItem.test.js [hunk 1]
```diff
@@ -4,11 +4,19 @@ import { render } from 'enzyme';
 import { DetailItem } from '../DetailItem';
 import { default as DetailItemFixtures } from '../fixtures/DetailItem.fixture';
 
-const testDetailItem = () => <DetailItem {...DetailItemFixtures[0].props} />;
+const testDetailItem = index => <DetailItem {...DetailItemFixtures[index].props} />;
 
 describe('<DetailItem />', () => {
   it('renders correctly', () => {
-    const component = render(testDetailItem());
+    const component = render(testDetailItem(0));
```
Comment (suomiy) [inline src/components/Dashboard/Details/tests/DetailItem.test.js]: can you please render all the fixtures in for cycle and use their name as a test identitier?

Comment (rawagner) [reply src/components/Dashboard/Details/tests/DetailItem.test.js]: done

**Comment by suomiy** [comment]
ci failed

**Comment by rawagner** [comment]
I had some obsolete snapshots - removed now. CI is passing

**Pull request merged**